WalletWasabi.Fluent/Models/UI/ApplicationSettings.cs

```diff
@@ -178,7 +178,8 @@ private void Save()
 			{
 				try
 				{
-					PersistentConfig newConfig = ApplyChanges(_startupConfig);
+					PersistentConfig currenctConfig = ConfigManagerNg.LoadFile<PersistentConfig>(_persistentConfigFilePath);
+					PersistentConfig newConfig = ApplyChanges(currenctConfig);
```

Comment:
Always the CurrentConfig (which is on the disk) needs to be taken into account, otherwise, the logic will work with outdated data.